pkg/controller/remote/cluster.go

```diff
@@ -0,0 +1,68 @@
+/*
+Copyright 2018 The Kubernetes Authors.
```

Comment:
2019

Reply:
Whoops, will fix tomorrow in a follow-up PR!